napari/window.py

```diff
@@ -0,0 +1,25 @@
+"""The Window class is the primary entry to the napari GUI.
+
+Currently, this module is just a stub file that will simply pass through the
+:class:`napari._qt.qt_main_window.Window` class.  In the future, this module
+could serve to define a window Protocol that a backend would need to implement
+to server as a graphical user interface for napari.
```

Comment:
Love this! Is there a reason for the except clause and dummy window right now?

Reply:
The main reason for the dummy window is providing a no-op `close()` function that would otherwise raise an unnecessary exception upon viewer.close() [here](https://github.com/tlambert03/napari/blob/676421a22ee2e8d02e922d7a657b1705d517c2f5/napari/viewer.py#L104)